core/services/ocr2/plugins/directrequestocr/scripts/gen_oracle_contract_config.go

```diff
@@ -25,35 +37,35 @@ func addrArrayToPrettyString(addrs []string) string {
 func main() {
 	// NOTE: replace values below with actual keys fetched from DON members
 	offchainPubKeysHex := []string{
-		"7ec48c19696fa23bc5706921e21e1a51423f24ddd0a9d0cce127d5ca0f76cb01",
```

Comment:
Can you actually leave those sample values intact? Or you can use the same one 4 times as an example. There is some benefit of showing how a given key/address looks like. For example, peerIDs always start with a "1", addresses need to have a "0x" prefix, etc.